models/experimental/mamba/benchmarks/README.md

```diff
@@ -0,0 +1,23 @@
+# lm-eval-harness.py
+
+To run zero-shot evaluations of our model (to compare against against to Table
```

Comment:
Please reword `to compare against against to`; probably should be `to compare against`.